CONTRIBUTING.md

```diff
@@ -31,5 +31,5 @@ Please keep this in mind if you dislike a possibility of a license change or, fo
 
 #### Pull requests
 
-Please raise all pull requests against `edge` instead of `master`.  

```

Comment:
@teo-tsirpanis I merged the UTF-8 fix (thanks!), but I am going to drop this specific line change as spaces are intentional -- they force the line break.